--- 4.stdout	2015-10-14 13:39:48.828753886 -0500
+++ 4.student.stdout	2015-10-14 13:48:39.481952423 -0500
@@ -1,8 +1,8 @@
-One When I had laid it on the floor
-Two I went to blow the fire aflame
-Three But something rustled on the floor
-Four And someone called me by my name
-Five It had become a glimmering girl
-Six With apple blossom in her hair
-Seven Who called me by my name and ran
+When I had laid it on the floor
+I went to blow the fire aflame
+But something rustled on the floor
+And someone called me by my name
+It had become a glimmering girl
+With apple blossom in her hair
+Who called me by my name and ran
 And faded through the brightening air
